frontend/src/components/OperationsTableCard/OperationsTableCard.tsx

```diff
@@ -143,6 +143,24 @@ const OperationsTableCard: FunctionComponent<OperationsTableCardProps> = (props)
     props.history.push(`/queries/build/${operation.get('id') as number}/`);
   };
 
+  const onUpdateOperation = (operation: OperationMap) => {
+    props.actions.setOperation(operation, true);
+  };
+
+  const onDuplicateOperation = (operation: OperationMap) => {
+    onUpdateOperation(operation);
+    props.history.push('/queries/build/');
+  };
+
+  const onDuplicate = (operation: OperationMap) => {
+    if (operation) {
+      const operationTwo = operation.withMutations((opn) =>
```

Comment:
rename `operationTwo` to `duplicateOperation`

Reply:
Oh forgot to change that